DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Ranking/Index.cshtml

```diff
@@ -19,24 +18,67 @@
 }
 
 @section NavBreadcrumbs {
-  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml"/>
+  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml" />
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
     <h1 class="nhsuk-heading-xl">Centre ranking</h1>
 
     <p class="nhsuk-body-l">The top ten Digital Learning Solutions centres by learner course launches.</p>
+  </div>
+</div>
 
-    @if (User.HasSuperAdminPermissions()) {
-      <partial name="_RankingSuperAdminTable" model="Model" />
-    } else {
-      <partial name="_RankingStandardUserTable" model="Model" />
-    }
+<div class="nhsuk-grid-row nhsuk-u-margin-bottom-4">
+  <form method="get" asp-controller="Ranking" asp-action="Index">
+    <div class="nhsuk-grid-column-one-quarter">
+      <vc:select-list asp-for="RegionId"

```

Comment:
I wonder if instead of referencing a string, we want to reference the model property here (to avoid errors due to typos etc) 🤔